services/app-web/src/pages/App/App.tsx

```diff
@@ -35,9 +36,14 @@ function App({
     apolloClient: ApolloClient<NormalizedCacheObject>
     s3Client: S3ClientT
 }): React.ReactElement {
-    useEffect(() => {
-        logEvent('on_load', { success: true })
```

Comment:
Can you check with Mojo to make sure we don’t need this anymore? I’m not sure if we are logging an event on load through otel yet.

Reply:
Let me just add that line back, I think we can clear out logEvent later I was getting hasty